boards/shields/silabs_wifi_expansion_kit/Kconfig.defconfig

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 grandcentrix GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_SILABS_WIFI_EXPANSION_KIT_SLEXP8023A || SHIELD_SILABS_WIFI_EXPANSION_KIT_SLEXP8022A
+
+if NETWORKING
+
+config SPI
+	default y
```

Comment:
redundant

Reply:
Hmm, should i add an `depends on SPI` to `config WIFI_WFX200` instead?